lighthouse-core/audits/seo/hreflang.js

```diff
@@ -121,9 +121,9 @@ class Hreflang extends Audit {
         source = {
           type: 'node',
           snippet: `<link rel="alternate" hreflang="${link.hreflang}" href="${link.hrefRaw}" />`,
-          path: link.devtoolsNodePath || '',
-          selector: link.selector || '',
-          nodeLabel: link.nodeLabel || '',
+          path: link.node !== null ? link.node.devtoolsNodePath : '',
```

Comment:
in reality this'll never get hit. in the gatherer only `source === 'headers'` can have a null `node`.

but.. writing it like this SGTM!  thanks TS!